specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-11-01/consumption.json

```diff
@@ -0,0 +1,5397 @@
+          "description": "Lot source.",
+          "type": "string",
+          "enum": [
+            "PurchasedCredit",
+            "PromotionalCredit"
+          ],
+          "x-ms-enum": {
+            "name": "LotSource",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "startDate": {
+          "description": "Start date.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "expirationDate": {
+          "description": "Expiration date.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "poNumber": {
+          "description": "PO number.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CreditSummary": {
+      "description": "A credit summary resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CreditSummaryProperties",
+          "title": "Credit summary properties"
+        }
+      }
+    },
+    "CreditSummaryProperties": {
+      "description": "The properties of the credit summary.",
+      "type": "object",
+      "properties": {
+        "balanceSummary": {
+          "description": "Summary of balances associated with this credit summary.",
+          "readOnly": true,
+          "$ref": "#/definitions/CreditBalanceSummary"
+        },
+        "pendingCreditAdjustments": {
+          "description": "Pending credit adjustments.",
+          "readOnly": true,
+          "$ref": "#/definitions/amount"
+        },
+        "expiredCredit": {
+          "description": "Expired credit.",
+          "readOnly": true,
+          "$ref": "#/definitions/amount"
+        },
+        "pendingEligibleCharges": {
+          "description": "Pending eligible charges.",
+          "readOnly": true,
+          "$ref": "#/definitions/amount"
+        }
+      }
+    },
+    "CreditBalanceSummary": {
+      "description": "Summary of credit balances.",
+      "properties": {
+        "estimatedBalance": {
+          "description": "Estimated balance.",
+          "readOnly": true,
+          "$ref": "#/definitions/amount"
+        },
+        "currentBalance": {
+          "description": "Current balance.",
+          "readOnly": true,
+          "$ref": "#/definitions/amount"
+        }
+      }
+    },
+    "amount": {
+      "description": "The amount plus currency .",
+      "properties": {
+        "currency": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Amount currency."
+        },
+        "value": {
+          "readOnly": true,
+          "type": "number",
+          "format": "decimal",
+          "description": "Amount."
+        }
+      }
+    },
+    "Resource": {
```

Comment:
Please reference the standard resource definitions here: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json#L9  You will likely use either TrackedResource or ProxyResource

Reply:
So in our definition ProxyResource has eTags, and in the common one it does not,  also for Resource we have defined tags + etags.  If you think this is ok I would like to keep it as it is for now.

Reply:
I just saw that there are some validation errors because of this, I will work on it. thanks!

Reply:
I don't see any changes related to this.

Reply:
I did take a look but did not update the comment, sorry about that,  I was looking for example UsageDetail, and it has tags, however it does not have a location currently in the response, this will end up with several errors in case to use TrackedResource (most of them falls in the same category).

It will required code changes from several teams,  Mark, do you think we can deliver this one as it is in order to get the completeness part, then we can ask each team to see how to move to the correct Resources definitions?